clients/ts/FunctionalTests/rollup.config.js

```diff
@@ -8,7 +8,7 @@ import commonjs from 'rollup-plugin-commonjs'
 import resolve from 'rollup-plugin-node-resolve'
 
 export default {
-    input: path.join(__dirname, "obj", "js", "index.js"),
+    input: path.join(__dirname, "obj", "js", "FunctionalTests", "ts", "index.js"),
```

Comment:
This is needed because now that the tests are importing from the `Utils.ts` file **inside** `@aspnet/signalr`, TypeScript changed the output folder structure.